gddragonbones.cpp

```diff
@@ -244,6 +251,15 @@ void GDDragonBones::set_resource(Ref<GDDragonBones::GDDragonBonesResource> _p_da
         __p_tad->width = m_texture_atlas->get_width();
     }
 
+	/* This here is what you call a hack for the end-user. Fix should be made in COA Tools; but it's put here for a quick win user-side*/
```

Comment:
To elaborate on this; @davord on Discord has been leading the charge in finding a good COA Tools -> DB flow and has found that armatures exported by COA Tools are coming out flipped on the y axis. This detects coa tools import as a convenience and flips the armature on import